rmgpy/reaction.py

```diff
@@ -613,6 +627,44 @@ def getRateCoefficient(self, T, P=0):
         else:
             return self.kinetics.getRateCoefficient(T, P)
 
+    def getSurfaceRateCoefficient(self, T, surfaceSiteDensity):
+        """
+        Return the overall surface rate coefficient for the forward reaction at
+        temperature `T` in K with surface site density `surfaceSiteDensity` in mol/m2.
+        Value is returned in combination of [m,mol,s]
+        """
+        cython.declare(rateCoefficient=cython.double, 
+                       molecularWeight_kg=cython.double, )
+
+        if diffusionLimiter.enabled:
+            raise NotImplementedError()
+        if not self.isSurfaceReaction():
+            raise ReactionError("This is not a surface reaction!")
+
+        if isinstance(self.kinetics, StickingCoefficient):
+            rateCoefficient= self.kinetics.getStickingCoefficient(T)
+            adsorbate = None
+            for r in self.reactants:
+                if r.containsSurfaceSite():
+                    rateCoefficient /= surfaceSiteDensity
+                else:
+                    adsorbate = r
```

Comment:
I am adding a check here to make sure to throw an error if there is more than one adsorbate